src/plugins/workspace/public/components/data_source_association/data_source_association.test.tsx

```diff
@@ -75,6 +75,13 @@ describe('<DataSourceAssociation />', () => {
       copy: jest.fn(),
       dissociate: jest.fn(),
       ui: jest.fn(),
+      getCurrentWorkspaceId: jest.fn(),
+      getCurrentWorkspace: jest.fn(),
+      create: jest.fn(),
+      delete: jest.fn(),
+      list: jest.fn(),
+      get: jest.fn(),
+      update: jest.fn(),
```

Comment:
Should we refactor this part by using https://github.com/opensearch-project/OpenSearch-Dashboards/blob/ae7cfa6072957ac2068807618a02804321731c2f/src/plugins/workspace/public/workspace_client.mock.ts ?